openshift/scripts/oc_deploy_to_production.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh -l
+#
```

Comment:
Let's also add `set -ex` to the top of the script so that if any command fails, the exit code is set to error and the pipeline will fail. This will notify us of errors. Currently our scripts ignore errors.

https://explainshell.com/explain?cmd=set+-ex